splitgpg2/__init__.py

```diff
@@ -901,6 +927,11 @@ def open_stdinout_connection(*, loop=None):
     return reader, writer
 
 def main():
+    # request bi-directional socket on stdin
+    server_pid = os.environ.get('QREXEC_AGENT_PID', '')
+    if server_pid:
+        os.kill(int(server_pid, 10), signal.SIGUSR1)
```

Comment:
I think you missed the ["server: Use stdin/-out instead of bi-directional socket" commit](https://github.com/HW42/qubes-app-linux-split-gpg2/commit/577fe177072f04605544387a7b11aafda6dda8fc) when rebasing.

Reply:
Now removed.